packages/loaders/src/elements/Dots.tsx

```diff
@@ -21,20 +21,19 @@ const COMPONENT_ID = 'loaders.dots';
 
 interface IDotsProps extends React.HTMLAttributes<SVGSVGElement> {
   /**
-   * Size of the loader. Can inherit from `font-size` styling.
+   * Sets the height and width in pixels. Inherits the parent `font-size` by default
    **/
   size?: string | number;
   /**
-   * Color of the loader. Can inherit from `color` styling.
+   * Sets the fill color
    **/
   color?: string;
   /**
-   * Duration (ms) of the animation. Default is 1250ms.
+   * Increases or decreases the speed of the loading animation in milliseconds
```

Comment:
Alternatively could be something along the lines of _"Sets a new loading animation duration in milliseconds"_. I think from a dev perspective this would be more accurate, since adding a duration prop is replacing the default duration value and not mutating it.

Reply:
For me, what I found most interesting about this prop after fiddling is that it's not actually changing the total duration of time the loader runs, but rather how fast or slow the loader animates once. This is tricky cause I realize `duration` from a css perspective is how fast or slow it takes to complete once, but from a plain language perspective I thought it specified how long it would run in general. 

Maybe: 
1. Sets the speed of the loading animation in milliseconds
2. Sets the time to complete one loading animation in milliseconds

Reply:
Went with 1, lemme know if we should give it another go

Reply:
Oh yeah that (1) sounds good!